d2layouts/d2elklayout/layout.go

```diff
@@ -96,7 +96,7 @@ var DefaultOpts = ConfigurableOpts{
 }
 
 var port_spacing = 40.
-var edge_node_spacing = 40
+var edge_node_spacing = 60
```

Comment:
isn't this just arbitrary? what if a larger label comes along?

Reply:
it is arbitrary, as the default is 10: https://www.eclipse.org/elk/reference/options/org-eclipse-elk-spacing-edgeNode.html -- is 40 not arbitary?

Reply:
yeah it's the arbitrariness that caused the bug in the first place though, we're trying to fix it and not make it arbitrary

Reply:
OK I'll poke around some more and see what I can suggest.